aruco_detect/scripts/create_markers.py

```diff
@@ -1,11 +1,18 @@
 #!/usr/bin/python
 
 import os, sys, argparse
-import em
+import subprocess
+import imp, importlib
+
+# Hack if a user has 'em' instead of 'empy' installed with pip
+# If the module em doesn't have the expand function use the full path
+em = importlib.import_module('em')
+if "expand" not in dir(em):
+    print "Using system empy because em is installed"
+    em = imp.load_source('em', '/usr/lib/python2.7/dist-packages/em.py')
```

Comment:
Could we get rid of this hard-coded path and determine it automatically?

Reply:
I don't think there is any way to find this path automatically, because the "em" name will be overloaded if the wrong pip package is installed.

Reply:
Could we use `sys.path`, eg 
```
for path in sys.path:
    filename = os.path.join(path, 'em.py')
    if os.path.exists(filename):
         print("Trying %s" % filename)
         em = imp.load_source('em', filename)
         if "expand" in dir(em):
             print("Has expand")
             break
```